app/components/search/Refinements/BrowseRefinementList.tsx

```diff
@@ -43,9 +43,11 @@ const BrowseRefinementList = ({ attribute, transform }: Props) => {
     setChecked(checked);
   };
 
+  const transformedItems = transform === undefined ? items : transform(items);
```

Comment:
**note:** @adriencyberspace ultimately I moved this transformation execution out of the algolia hook to fix the performance bottleneck and give our code . This works, the infinite looping is gone.